src/Raven.Server/Documents/Transformers/Transformer.cs

```diff
@@ -101,11 +101,11 @@ private void Persist()
             File.WriteAllText(GetPath(TransformerId, Name, _configuration).FullPath, JsonConvert.SerializeObject(Definition, Formatting.Indented));
```

Comment:
We are we writing this to disk anyway? This is now all stored in the database record on the server, no?